mcodingbot/plugins/bot_management.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import annotations
+
+import crescent
+import hikari
+
+from mcodingbot.config import CONFIG
+
+plugin = crescent.Plugin()
+
+
+@plugin.include
+@crescent.command(
+    name="restart",
+    description="Restarts the bot.",
+    default_member_permissions=hikari.Permissions.MODERATE_MEMBERS,
+    dm_enabled=False,
+    guild=CONFIG.mcoding_server,
+)
+class RestartBot:
+    async def callback(self, ctx: crescent.Context) -> None:
+        await ctx.respond("Restarting bot...")
+        await ctx.app.close()
```

Comment:
```suggestion
async def restart_bot(ctx: crescent.Context) -> None:
    await ctx.respond("Restarting bot...")
    await ctx.app.close()
```

Reply:
I did this to be consistent

https://github.com/mcb-dev/mCodingBot/blob/main/mcodingbot/plugins/info.py#L10

We can change this in a seperate pr if we want to.

Reply:
I feel like that one should be changed. I want to give function commands some love :(
Links would benefit a bit form a lower indentation level as well

Reply:
If it's going to be changed I think it should be done in a new pr, since so far every command (even ones without options) are class commands.

Reply:
:+1: